tests/test_authtoken.py

```diff
@@ -21,6 +22,14 @@ def setUp(self):
         self.user = User.objects.create_user(username='test_user')
         self.token = Token.objects.create(key='test token', user=self.user)
 
+    def test_authtoken_can_be_imported_when_not_included_in_installed_apps(self):
+        import rest_framework.authtoken.models
+        with modify_settings(INSTALLED_APPS={'remove': 'rest_framework.authtoken'}):
+            importlib.reload(rest_framework.authtoken.models)
+        # Set the proxy and abstract properties back to the version,
+        # where authtoken is among INSTALLED_APPS.
+        importlib.reload(rest_framework.authtoken.models)
```

Comment:
you probably want to put the final reload in a `finally:` block, that way the imports aren't messed up for a subsequent test. I ran into this issue while adding tests to my PR.